src/components/founderDetail/OverviewBoxButtons.tsx

```diff
@@ -10,37 +10,59 @@ import { Button } from "@tremor/react";
 import { useState } from "react";
 import { remult } from "remult";
 import LinkButton from "../buttons/LinkButton";
+import { SchoolTypeValues, getSchoolTypeCode } from "@/entities/School";

```

Comment:
`SchoolTypeValues` and `PencilSquareIcon` are unused, you can remove unused imports by `Organize Imports` command in VS code (Shift+Alt+O on Windows).